cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -345,6 +352,13 @@ def _delete_project(self, project_id, user):
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
         return response
 
+    @staticmethod
+    def _save_file_from_response(response, file_zip_name):
```

Comment:
```suggestion
    def _save_file_from_response(response, file_name):
```

Doesn't seem to be anything zip-specific here.

Reply:
renamed